src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -610,7 +610,8 @@ enum SQLServerDriverStringProperty {
     ENCRYPT("encrypt", EncryptOption.TRUE.toString()),
     SERVER_CERTIFICATE("serverCertificate", ""),
     DATETIME_DATATYPE("datetimeParameterType", DatetimeType.DATETIME2.toString()),
-    ACCESS_TOKEN_CALLBACK_CLASS("accessTokenCallbackClass", "");
+    ACCESS_TOKEN_CALLBACK_CLASS("accessTokenCallbackClass", ""),
+    RETRY_EXEC("retryExec", "");
```

Comment:
did David ok this name? seems a bit too vague to me I would think something more descriptive would be better as all our other properties are more descriptive